model/semconv/v1.6.1/nonstandard.go

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semconv
+
+const (
+	OtelLibraryName       = "otel.library.name"
+	OtelLibraryVersion    = "otel.library.version"
+	OtelStatusCode        = "otel.status_code"
+	OtelStatusDescription = "otel.status_description"
+)
+
+const (
+	AttributeMessageType    = "message.type"
+	AttributeHTTPStatusText = "http.status_text"
+)
+
+const (
+	ExceptionEventName = "exception"
```

Comment:
Same comment this probably needs to be added when semantic conventions in specs can generate it.